README.md

```diff
@@ -155,7 +155,21 @@ $ make clean
 
 ### Building with Doxygen
 
-If you want to build the Pico C SDK Doxygen documentation alongside the main documentation site you can do so with,
+If you want to build the Pico C SDK Doxygen documentation alongside the main documentation site, first install the following dependencies:
+
+NOTE: For the time being, building with any Doxygen version other than 1.8.17 will result in formatting errors in the built site.
+
+```
+$ brew install doxygen@1.8.17 graphviz
+```
+
+Or on Linux, 
+
+```
+$ sudo apt -y install doxygen=1.8.17 graphviz
```

Comment:
It's worth noting this isn't available from the default set of apt repositories on the latest version of Raspberry Pi OS where only Doxygen 1.9.4 is available:

```
$ apt policy doxygen
doxygen:
  Installed: (none)
  Candidate: 1.9.4-4
  Version table:
     1.9.4-4 500
        500 http://deb.debian.org/debian bookworm/main arm64 Packages
```